modules/test/playwright/tests/batch-planner/export.spec.ts

```diff
@@ -0,0 +1,408 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+import * as fs from 'fs';
+import * as path from 'path';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {dataMigrationCenterPagesTest} from '../../fixtures/dataMigrationCenterPages';
+import {exportImportPagesTest} from '../../fixtures/exportImportPages.fixtures';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {streamToJson, unzipFile} from '../../utils/zipFolder';
```

Comment:
we are mixing too many different functions in a module called "zipFolder". Either we rename the module to something else (zip??) or we split in different files.